src/js/tech/html5.js

```diff
@@ -962,6 +1079,27 @@ Html5.supportsNativeTextTracks = function() {
   return supportsTextTracks;
 };
 
+/*
+ * Check to see if native video tracks are supported by this browser/device
+ *
+ * @return {Boolean}
+ */
+Html5.supportsNativeVideoTracks = function() {
```

Comment:
#3166 has a nice method that took a string `text`, `audio`, or `video`, and returned whether that thing is supported. Would be good to cherry-pick that change, remembering that `supportsNativeTextTrack` needs to stay for backwards compat but it could be deprecated.